src/components/CardMenus/Tabs.js

```diff
@@ -5,7 +5,7 @@ import { Paper, Tab, Tabs } from '@material-ui/core';
 import CardMenus from './CardMenus';
 
 const MainTab = () => {
-  const [value, setValue] = useState(2);
+  const [value, setValue] = useState();
```

Comment:
is there any way to handle the tab instead of using this state variable, i think it is not appropriate, what if we handle it through react-router a separate URL for each screen.